### PR TITLE
if else block static check + unit tests

### PR DESCRIPTION
**Pull request by lchsam**
- `if else` static check

### CODE REVIEW

File: test/unit-tests.test.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { compile } from '../ts/index';
 import { sandbox } from '../ts/sandbox';
 import * as runtime from '../ts/runtime';
 import { default as generator } from 'babel-generator';
+import 'jest'
```
Comment (joydeep-b) [inline test/unit-tests.test.ts]: Why is this required?

File: ts/visitor.ts [hunk 2]
```diff
@@ -401,6 +401,15 @@ export const visitor = {
       st.elem.error(path, `Loop body must be enclosed in braces.`);
     }
   },
+  IfStatement(path: NodePath<t.IfStatement>, st: S) {
+    if (!t.isBlockStatement(path.node.consequent) && path.node.alternate === null) {
+      st.elem.error(path, `if statement body must be enclosed in braces.`);
+      return;
+    }
+    if (!t.isBlockStatement(path.node.consequent) && !t.isBlockStatement(path.node.alternate)) {
+      st.elem.error(path, `Two bodies of if else statement must be enclosed in braces.`);
```
Comment (joydeep-b) [inline ts/visitor.ts]: The "two" is confusing. How about "Body of if-else statement must be enclosed in braces."

File: test/unit-tests.test.ts (hunk 1 above)
Comment (lchsam) [reply test/unit-tests.test.ts]: Oh sorry, I accidentally left it there. I'll remove it

File: ts/visitor.ts (hunk 2 above)
Comment (lchsam) [reply ts/visitor.ts]: Got it.
